examples/dynamic/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/kami-zh/go-capturer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(t *testing.T) {
+	require := require.New(t)
+	os.Args = []string{"test", "--help"}
+	var (
+		stdout, stderr string
+	)
+	stdout = capturer.CaptureStdout(func() {
+		stderr = capturer.CaptureStderr(func() {
```

Comment:
Why are these functions nested?

Reply:
@dennwc  What would you propose instead?

Reply:
Sorry, I thought that these variables are `io.Writer`s. Don't see a good alternative. It's still a bit weird, though.